internal/command/arguments/types.go

```diff
@@ -31,3 +35,9 @@ func (vt ViewType) String() string {
 		return "unknown"
 	}
 }
+
+type testFile struct {
```

Comment:
Given the `tofu test` command, we try to be careful about naming anything `test` in a non _test.go file.

Could this struct be moved to types_test.go?

Reply:
Could you point me at the `types_test.go` file you're talking about? The only one I can find is [here](https://github.com/opentofu/opentofu/blob/main/internal/getproviders/types_test.go) which I assume is not where this struct belongs.

I can also move it into `plan_test.go`, but as it's shared between all three tests (`apply_test.go`, `plan_test.go`, refresh_test.go`), I didn't think this was a good idea.

Reply:
I was suggesting you create a new file types_test.go, adjacent to types.go

Reply:
Ah I understand now.
I've moved it to a new file and renamed the struct to `mockFile` to avoid the `tofu test` confusion. I've also moved it's `tempFileWriter` method into the file with it. I believe that addresses your comment, so I'll resolve this comment for now.

Just a side note that this `types_test.go` filename seems a bit confusing to me though. I would expect a such a file to contain tests for exported functions from [the`types.go` file](https://github.com/opentofu/opentofu/blob/main/internal/command/arguments/types.go) (at the moment, this would only be for the String() method). So to put shared structs and methods for `*_test.go` files in here seems counterintuitive to me.